app/views/ListRadiosView.scala.html

```diff
@@ -31,7 +31,7 @@
     @formHelper(action = viewmodel.onSubmit, Symbol("autocomplete") -> "off") {
 
         @if(form.hasErrors) {
-          @govukErrorSummary(ErrorSummaryViewModel(form))
+          @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> viewmodel.page.rows.head.index.toString)))
```

Comment:
Can rows ever be empty?

Reply:
I don't think so, we should open this page only when there is data

Reply:
In case of empty list, it will show the whole component as it was.